stories/__snapshots__/AboutUs.stories.snapshot

```diff
@@ -1028,26 +1028,6 @@ exports[`Storyshots AboutUs Partners 1`] = `
 }
 
 .c6 {
-  -webkit-flex-wrap: wrap;
```

Comment:
Yeah, as we've talked about, I don't feel that most of the storyshots provide a lot of value in their current state :sweat_smile:

Reply:
ditto